components/builder-depot/src/server.rs

```diff
@@ -1006,6 +1049,7 @@ pub fn router(depot: Depot) -> Result<Chain> {
 
         package_search: get "/pkgs/search/:query" => search_packages,
         packages: get "/pkgs/:origin" => list_packages,
+        packages_unique: get "/pkgs-unique/:origin" => list_unique_packages,
```

Comment:
Yeah, I agree that this URL feels terrible.  My first cut at this was `/pkgs/:origin/unique` but then I realized that means no one could have a package named `unique` which seemed like a bad idea.  I like `/:origin/pkgs`.  Maybe that's the best way to go.  My only concern with doing it in a header is it makes things a bit more difficult to understand.  Now just having a URL isn't enough - you also need to know about the associated header.